src/files.cpp

```diff
@@ -0,0 +1,17 @@
+#include "files.h"
+
+void getFiles(std::string directory, std::vector<fs::path> &files) {
```

Comment:
This could even be a `const char*` instead of `std::string`, but of course you can construct a `std::string` with the former so you don't accomplish much by doing so.

Reply:
Added both a test case as well as changed the type from string to char*